ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -151,4 +150,10 @@ $defs:
           description: EOS CLI rendered directly on the point-to-point interface in the final EOS configuration.
         structured_config:
           type: dict
-          description: Custom structured config for interfaces
+          documentation_options:
+            hide_keys: true
+          description: |-
+            Custom structured config for interfaces
+            Note! The content of this dictionary is _not_ validated by the schema, since it can be either ethernet_interfaces or port_channel_interfaces.
```

Comment:
For future perhaps we have ethernet_structured_config and port_channel_structured_config?